collections/docs/01-intro.md

```diff
@@ -0,0 +1,232 @@
+# Collections
```

Comment:
This still needs to be colllapsed in one README.
What I mean is: 01-intro.md, 02-map.md, 03-keyset.md,... should be in one file: collections/README.md. Like we do for modules.

Reply:
should've been addressed, let me know!